src/compute-client/src/controller/instance.rs

```diff
@@ -262,6 +262,18 @@ impl<T> Instance<T> {
         };
         self.ready_responses.push_back(resp);
     }
+
+    /// List compute objects that depend on the given object.
+    pub fn reverse_compute_dependencies(&self, id: GlobalId) -> impl Iterator<Item = &GlobalId> {
+        self.collections_iter()
+            .filter_map(move |(dependant_id, state)| {
```

Comment:
```suggestion
            .filter_map(move |(id, state)| {
```

Nit: Not all of these collections are dependants, only the ones that pass the condition below.

Reply:
`collection_id` would have been my suggestion, but that works too!